docs/Templates/SpikeOutcome-Template.md

```diff
@@ -2,53 +2,40 @@
 
 ==================
 
-**Spike:** Spike_No
+**Spike:** Splashkit_camera
 
-**Title:** Spike_Title
+**Title:** Test Python Version
 
-**Author:** John Doe, john.doe@example.com
+**Author:** Darren Marchiano Sunandar
 
-## Goals / Deliverables
-
-Summarise from the spike plan goal* Besides this report, what else was created ie UML, code,
-reports*
-
-- Code see /spikes/spike04/
-- Short Report titled “IDE Comparison”
 
 ## Technologies, Tools, and Resources used
 
-List of information needed by someone trying to reproduce this work\
-
 - Internet Browser; Google Chroame, FireFox, Safari
 - Programming Languages:
-- Ruby
+- Python
 - Programming Libraries:
-- Minitest
+- Python libraries
 - Text Editor: VsCode
 - Terminal
 
 ## Tasks undertaken
 
 List key tasks likely to help another developer
 
-- ...
+- I've tested all the python function from the Splashkit website guides such as Animations, Audio, Camera, Graphics, etc. And there is 1 error,
+- From the Splashkit camera guides. In line 49 fill_rectangle() takes 5 positional arguments but 6 were given.
 
 ## What we found out
 
-Describe (sentences), + graphs/screenshots/outcomes as needed
-
-## Open issues/risks _[Optional – remove heading/section if not used!]_
-
-List out the issues and risks that you have been unable to resolve at the end of the spike. You may
-have uncovered a whole range of new risks as well. Make notes to help the team manage and respond.\*
-
-- e.g. Risk xyz (new)
+![alt text](image.png)
 
 ## Recommendations _[Optional – remove heading/section if not used!]_
 
 Often based on any open issues/risks Identified. You may state that another spike is required for
 the team to resolve new issues identified (or) indicate that this spike has increased the teams
 confidence in XYZ and should move on.\*
```

Comment:
"_[Optional – remove heading/section if not used!]_" should be removed along with the placeholder text underneath.